Crashlytics/Crashlytics/Rollouts/CrashlyticsRemoteConfigManager.swift

```diff
@@ -81,4 +99,31 @@ private extension CrashlyticsRemoteConfigManager {
 
     return validatedAssignments
   }
+
+  func getRolloutAssignmentsEncodedJsonData() -> Data? {
```

Comment:
Can we add a comment explaining why we're using NSData?

Reply:
done!